NEWS.md

```diff
@@ -17,7 +17,8 @@
   
 - A bug was fixed where the initial growth was never estimated (i.e. the prior mean was always zero). By @sbfnk in #853 and reviewed by @seabbs.
 - A bug was fixed where an internal function for applying a default cdf cutoff failed due to a difference a vector length issue. By @jamesmbaazam in #858 and reviewed by @sbfnk.
-- All parameters have been changed to the new parameter interface. By @sbfnk in #871 and reviewed by @seabbs.
+- All parameters have been changed to the new parameter interface. By @sbfnk in #871 and #890 and reviewed by @seabbs.
+- The Gaussian Process lengthscale is now scaled by half the length of the time series. By @sbfnk in #890 and reviewed by #
```

Comment:
This doesn't say what it was beefore and I think it should. I am also really not convinced this is a sensible choice as in most cases we would think about the lengthscale as being driven by the epi process and not the amount of data being fit (it will also really worse the difference we see between longer and shorter fits to data as the prior will change a lot)